angular-ngrx-scss/src/app/shared/components/issue-pull-item/issue-pull-item.component.ts

```diff
@@ -0,0 +1,50 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+
+@Component({
+  selector: 'app-issue-pull-item',
+  templateUrl: './issue-pull-item.component.html',
+  styleUrls: ['./issue-pull-item.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class IssuePullItemComponent {
+  itemsList = [
```

Comment:
suggestion: Let's make sure this is set up to work with provided info, and not just this hard coded list! Then when this is cleaned up once everything else is ready, it should "just work" when given any list. :)